assets/js/screens/BatchEdit/BatchEdit.jsx

```diff
@@ -3,7 +3,608 @@ import Layout from "../Layout";
 import UIBreadcrumbs from "../../components/UI/Breadcrumbs";
 import BatchEditPreviewItems from "../../components/BatchEdit/PreviewItems";
 import BatchEditTabs from "../../components/BatchEdit/Tabs";
-
+const selectedItemsMock = [
+  {
+    accessionNumber: "Xample-b-25",
+    collection: null,
+    descriptiveMetadata: {
+      caption: [],
+      identifier: [],
+      title: "Sample Title 1",
+    },
+    id: "0239eeb9-1763-4c78-8948-853365e487dc",
+    representativeImage:
+      "https://devbox.library.northwestern.edu:8183/iiif/2/ed15d61e-1e47-4b5e-8447-bb39fb007a46",
```

Comment:
These are local image references, so will appear broken to anyone else previewing. For now, can you link to a bulma test image?  https://bulma.io/documentation/elements/image/#responsive-images-with-ratios   And just do a full url reference like:

```
representativeImage:
      "https://bulma.io/images/placeholders/480x480.png",
...
```